ddl/etcd_worker.go

```diff
@@ -14,18 +14,43 @@
 package ddl
 
 import (
+	"fmt"
+	"sync"
 	"sync/atomic"
 	"time"
 
+	"github.com/coreos/etcd/clientv3"
 	"github.com/coreos/etcd/clientv3/concurrency"
 	"github.com/coreos/etcd/mvcc/mvccpb"
 	"github.com/juju/errors"
 	"github.com/ngaut/log"
 	goctx "golang.org/x/net/context"
 )
 
-// ChangeOwnerInNewWay is used for testing.
-var ChangeOwnerInNewWay = false
+// EtcdWorker is an interface for DDL access the etcd.
+type EtcdWorker interface {
```

Comment:
How about rename it to `DDLSyncer`.
DDL workers use DDLSyncer to  synchronize ddl information. Using etcd to synchronize ddl information is an implementation of DDLSyncer. So put `etcd` in the name of the interface is not proper.